adaptdl/adaptdl/speedup_test.py

```diff
@@ -50,27 +53,30 @@ def test_local_bounds():
         params = np.random.gamma(2.0, 2.0, (7,))
         grad_params = np.random.gamma(2.0, 2.0, (2,))
         grad_params = {"norm": grad_params[0], "var": grad_params[1]}
-        fun = SpeedupFunction(params, grad_params, 128, None, (64, 256), True)
-        speedup, bsz = fun(1, 1, return_local_bsz=True)
+        fun = SpeedupFunction(params, grad_params, 128, None, (64, 256),
+                              None, True)
+        speedup, (bsz, steps) = fun(1, 1, return_local_bsz=True)
         assert(bsz == 128), "expected bsz = 128, got {}".format(bsz)
         assert(isinstance(speedup, float))
 
         replicas = np.asarray(range(1, 100))
         # single-node
-        speedup, bsz = fun(np.ones_like(replicas), replicas,
-                           return_local_bsz=True)
+        speedup, (bsz, steps) = fun(np.ones_like(replicas), replicas,
+                                    return_local_bsz=True)
         assert(np.all(bsz >= np.ceil(128 / replicas).astype(int)))
         assert(np.all(np.logical_or(bsz >= (64), speedup == 0.0)))
         assert(np.all(bsz <= (256)))
         assert(np.all(bsz * replicas <= 100 * 128))
         assert(bsz[0] == 128)
+        assert(np.all(steps == 1))
         # multi-node
-        speedup, bsz = fun(replicas, replicas, return_local_bsz=True)
+        speedup, (bsz, steps) = fun(replicas, replicas, return_local_bsz=True)
```

Comment:
`return_local_bsz` parameter name is no longer accurate since it's returning both bsz and steps now. Maybe `return_config` instead?